packages/graph/hash_graph/lib/graph/src/knowledge/entity/mod.rs

```diff
@@ -266,13 +279,21 @@ impl Entity {
 impl Record for Entity {
     type EditionId = EntityEditionId;
     type QueryPath<'p> = EntityQueryPath<'p>;
+    type VertexId = EntityVertexId;
 
     fn edition_id(&self) -> &Self::EditionId {
         &self.metadata.edition_id
     }
 
-    fn create_filter_for_edition_id(edition_id: &Self::EditionId) -> Filter<Self> {
-        Filter::for_entity_by_edition_id(*edition_id)
+    fn vertex_id(&self) -> Self::VertexId {
+        EntityVertexId::new(
+            self.edition_id().base_id(),
+            self.metadata().version().transaction_time().start,
```

Comment:
Yes, this is going to be changed, but as there is no interface yet for the projection (#1738), this will be changed in #1743 to 
```rust
fn vertex_id(&self, time_axis: TimeAxis) -> Self::VertexId {
    EntityVertexId::new(self.edition_id().base_id(), match time_axis {
        TimeAxis::DecisionTime => self.metadata().version().decision_time().start.cast(),
        TimeAxis::TransactionTime => self.metadata().version().transaction_time().start.cast(),
    })
}
```